esx_service/vmci/vmci_client.c

```diff
@@ -291,6 +291,8 @@ vsock_get_reply(be_sock_id *s, be_request *r, be_answer* a)
    ret = recv(s->sock_id, a->buf, b, 0);
    if (ret == -1 || ret != b) {
       free(a->buf);
+      a->buf = NULL;
+      fprintf(stderr, "Failed to receive message data: received %d expected %d\n", ret, b);
```

Comment:
i think it goes to /dev/null as for any other daemon.  stderr was mainly intended for unit debugging and was never converted to syslog or other logging facility